database/src/migrations/20210611115350_block_observation.ts

```diff
@@ -0,0 +1,95 @@
+import Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+
+  GRANT ALL ON SCHEMA biohub TO postgres;
```

Comment:
Is this line needed? If this gets run by the previous migration, then I assume its already set up by the time this migration runs?

Reply:
Looking into it.

Reply:
not needed.